Library/Formula/google-java-format.rb

```diff
@@ -0,0 +1,18 @@
+class GoogleJavaFormat < Formula
+  desc "Tool to format java code with the google java format"
+  homepage "https://github.com/google/google-java-format"
+  url "https://github.com/google/google-java-format/releases/download/google-java-format-0.1-alpha/google-java-format-0.1-alpha.jar"
+  version "0.1-alpha"
+  sha256 "4baec0288c8d676100342c4a7beb08758428bbad005209732250710d7b46f71f"
+
+  bottle :unneeded
+
+  def install
+    libexec.install "google-java-format-#{version}.jar"
+    bin.write_jar_script libexec/"google-java-format-#{version}.jar", "google-java-format"
+  end
+
+  test do
+    system "#{bin}/google-java-format", "-v"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!